pydis_site/apps/api/migrations/0087_unique_constraint_filters.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0086_new_filter_schema'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
```

Comment:
How would I reverse it?

Reply:
It should be done by adding `reverse_sql="ALTER TABLE api_filter DROP CONSTRAINT unique_filters"` kwarg to the `RunSQL` instance.

The reason I'm asking this is because when testing locally, and then wanting to move to a different branch to test something else, you'll need to apply migrations up until some version prior to yours, which will result in a 
`django.db.migrations.exceptions.IrreversibleError`

This is ok to happen locally, as you can always drop the container & spin up a new one. 

It's also fair to day that I'm not sure this is something that'll ever happen in production, so we might see what others have to say about it before we action it.

Reply:
I added it anyway, it can't hurt [1385060](https://github.com/python-discord/site/pull/861/commits/138506045acd1cf464dc63d4465d36208801a3ab)

However if you need to switch to another branch then I'm not sure if some of the other migrations are reversible. I use a separate clone for this project.

Reply:
I was referring to all the filter migrations, but I don't think this'll ever be a concern.
Thanks for doing this already, and i think it's fine to skip the rest.